mode/sql/ssms.html

```diff
@@ -0,0 +1,73 @@
+<!doctype html>
```

Comment:
This doesn't need to be part of the pull request—there's no need to have a demo page for every dialect.